packages/core/src/components/block/block.tsx

```diff
@@ -1,7 +1,12 @@
 import { Component, h, Prop, Element } from '@stencil/core';
 
 /**
- * @slot <default> - <b>Unnamed slot.</b> For the content.
+ * @slot - Default slot for content inside the block.
+ *
+ * @example
+ * <tds-block>
+ *   <section>Semantic section content</section>
+ * </tds-block>
```

Comment:
```suggestion
 * @slot - <default> - <b>Default</b> slot for content inside the block.
 *
 * @example
 * <tds-block>
 *   <section>Semantic section content</section>
 * </tds-block>
```

Otherwise this is how auto generated docs looks like:
![Screenshot 2025-03-05 at 15 35 45](https://github.com/user-attachments/assets/ebbdffc4-9c17-4486-8096-05ad91f8db8d)